src/modules/tests/test-data/cat-c/manoeuvres/__tests__/manoeuvres.cat-c.selectors.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { CatCUniqueTypes } from '@dvsa/mes-test-schema/categories/C';
+import { getReverseLeftSelected } from '../manoeuvres.cat-c.selectors';
+
+describe('Manoeuvres CatBE Selectors', () => {
```

Comment:
CatBE in the describe... should be cat c

Reply:
changed it